README.md

```diff
@@ -3,7 +3,13 @@
 ## Getting Started
 
 ### Install dependencies
+```bash
+$ npm install
+# or
+$ yarn install
```

Comment:
As for the purpose of keeping all locked versions of dependencies consistent, we only use npm install. Please remove the yarn install line. Otherwise, all good.